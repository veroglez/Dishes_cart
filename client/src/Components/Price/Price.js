import React, { PureComponent } from 'react';
import { number } from 'prop-types';
import priceFormat from '../../common/priceFormat';
import style from './Price.module.scss';

class Card extends PureComponent {
  static propTypes = {
    price: number.isRequired,
  };

  render() {
    const { props: { price } } = this;

    return (
      <p className={style.price}>{priceFormat(price)}</p>
    );
  }
}

export default Card;
