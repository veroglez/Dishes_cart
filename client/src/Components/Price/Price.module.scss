.price {
  font-weight: 500;
}