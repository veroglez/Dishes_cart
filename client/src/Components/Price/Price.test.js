import React from 'react';
import renderer from 'react-test-renderer';
import Price from './Price';

describe('<Price>', () => {
  it('render', () => {
    const tree = renderer.create(<Price price={10} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
