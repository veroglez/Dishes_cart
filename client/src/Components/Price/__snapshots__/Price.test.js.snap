// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Price> render 1`] = `
<p
  className="price"
>
  0.1 â‚¬
</p>
`;
